### PR TITLE
JSTL - Student Message of the Day

### PR DESCRIPTION
**Pull request by MrFool**
@taniach could you take a look at this, you're going to need it for your PRs at #3754 and #3760

### CODE REVIEW

**Comment by MrFool** [comment]
Take Note:

I fixed this in the process so when you use this you might need to manually change the HTML, not 100% sure but still take note anyway.

screen shot 2015-06-25 at 3 39 44 pm

The class = was not there but I inserted it in this PR.

File: src/main/webapp/WEB-INF/tags/student/studentMessageOfTheDay.tag [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<%@ tag description="Student Message of the day" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Config" %>
+<c:set var="motdUrl" value="<%= Config.inst().STUDENT_MOTD_URL %>" />
+<c:if test="${motdUrl != null && not empty motdUrl}">
```
Comment (taniach) [inline src/main/webapp/WEB-INF/tags/student/studentMessageOfTheDay.tag]: not empty motdUrl will check that it is not null

File: src/main/webapp/WEB-INF/tags/student/studentMessageOfTheDay.tag [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            },
+            error: function(jqXHR, textStatus, errorThrown) {
+                console.log('AJAX request failed');
+            }
+        });
+
+        function closeMotd() {
+            $('#student-motd-container').hide();
+        }
+    </script>
```
Comment (taniach) [inline src/main/webapp/WEB-INF/tags/student/studentMessageOfTheDay.tag]: Minor change - line break after script

File: src/main/webapp/WEB-INF/tags/student/studentMessageOfTheDay.tag (hunk 1 above)
Comment (MrFool) [inline src/main/webapp/WEB-INF/tags/student/studentMessageOfTheDay.tag]: ohh, ok, I'll get to that in a bit, didn't know that it'll check for not null too!

**Comment by MrFool** [comment]
Updated

**Comment by taniach** [comment]
Can merge it

**Pull request merged**